publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/restrict/NonDegreePermissionStrategy.java

```diff
@@ -15,7 +15,7 @@ public NonDegreePermissionStrategy(Publication publication, UserInstance userIns
 
     @Override
     public boolean deniesAction(PublicationOperation permission) {
-        if (isUsersDraft()) {
+        if (isUsersDraft() || userInstance.isExternalClient()) {
```

Comment:
Should there be a check to verify that the user has something to do with the publication? Like `isUsersDraft()` does? Now, a third party can access any degree. Unless that is the intention tho.

Reply:
I'd verify with PO that the scope is "own publications", where "own" needs to be defined

Reply:
This change does not allow anything, its just make sure that an external client is not stopped. This is a restrict policy and external clients are today mostly handled on scope and API level granularity, not really doing much with clients inside the strategies.

Reply:
From your other comment I can see now that third parties without the correct credentials will be "stopped" by the ThirdPartyStrategy  👍  A little bit unfortunate that third parties has to be taken care of here as well, but I can't see another way atm.